src/app/services/auth.service.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpClient } from '@angular/common/http';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
import { HttpClient } from "@angular/common/http";
```